libopenage/renderer/camera/camera.h

```diff
@@ -23,6 +25,13 @@ class UniformBuffer;
 
 namespace camera {
 
+/**
+ * Defines constant boundaries for the camera's view in the X and Z axes.
+ */
+struct CameraBoundaries {
+	float x_min, x_max, z_min, z_max;
```

Comment:
I think it's also a good idea to add `y_min` and `y_max` for the future, even if we don't use them right now.